<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Текстовый редактор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
    <item>
     <layout class="QHBoxLayout" name="hLayout" stretch="5">
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="newFileAction"/>
    <addaction name="openFileAction"/>
    <addaction name="readOnlyAction"/>
    <addaction name="saveFileAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="showHelpAction"/>
    <addaction name="aboutProgramAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="shortcutsPrefsAction"/>
    <addaction name="translationChangeAction"/>
    <addaction name="changeThemeAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="openFileAction">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="saveFileAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="showHelpAction">
   <property name="text">
    <string>Показать справку</string>
   </property>
  </action>
  <action name="aboutProgramAction">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="readOnlyAction">
   <property name="text">
    <string>Открыть для чтения</string>
   </property>
  </action>
  <action name="newFileAction">
   <property name="text">
    <string>Новый</string>
   </property>
   <property name="iconText">
    <string>Новый</string>
   </property>
  </action>
  <action name="shortcutsPrefsAction">
   <property name="text">
    <string>Клавиши быстрого доступа</string>
   </property>
  </action>
  <action name="translationChangeAction">
   <property name="text">
    <string>Сменить язык интерфейса</string>
   </property>
  </action>
  <action name="changeThemeAction">
   <property name="text">
    <string>Сменить тему</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
